### ISSUE TITLE
Latest OF master version?

### ISSUE THREAD

**Issue by fabiaserra** [genekogan/ofxSecondWindow#4]
I love this addon!! It makes it very simple to work with two windows. With the latest version of openframeworks from the github I can't get it to work though... Any ideas why? I don't get any error, I guess it is just the GLFW updates?

Thanks!!

**Comment by genekogan** [comment]
i haven't checked it yet, but with the latest version of OF (and in the upcoming release), multi-window should be natively supported, making this addon (soon to be) obsolete.

**Comment by fabiaserra** [comment]
Yes... there are some examples of multi-window from Arturo but the workflow is more complex than this with your addon and is hard to use with ofxUI and to share resources between windows.

Anyway, I'll keep checking the new releases and use the stable version instead if I don't find a workaround.

Thanks again!

**Comment by javl** [comment]
To add to this: I don't think the new system in OF 0.9 can create undecorated windows, making your addon still very useful. 

(I found it while looking for an alternative to the OF second screen but I actually need both windows to be undecorated, something your addon doesn't seem to be able to do either).

**Comment by genekogan** [comment]
sure it can.  just use `setFullscreen(true)`.  i dont have two monitors in front of me so can't test this at the moment, but pretty sure you can do it with both windows.

```
#include "ofMain.h"
#include "ofApp.h"
#include "ofAppGLFWWindow.h"

//========================================================================
int main( ){
    ofGLFWWindowSettings settings;
    settings.width = 600;
    settings.height = 600;
    settings.setPosition(ofVec2f(300,0));
    settings.resizable = true;
    shared_ptr<ofAppBaseWindow> mainWindow = ofCreateWindow(settings);

    settings.width = 300;
    settings.height = 300;
    settings.setPosition(ofVec2f(0,0));
    settings.resizable = false;
    shared_ptr<ofAppBaseWindow> guiWindow = ofCreateWindow(settings);
    guiWindow->setVerticalSync(false);
    guiWindow->setFullscreen(true);

    shared_ptr<ofApp> mainApp(new ofApp);
    mainApp->setupGui();
    ofAddListener(guiWindow->events().draw,mainApp.get(),&ofApp::drawGui);

    ofRunApp(mainWindow, mainApp);
    ofRunMainLoop();

}

```

**Comment by javl** [comment]
I'm afraid that won't work for my situation. I have one image across two projectors (1920 x 2 by 1080) and one across four standing monitors (1080 x 4 by 1920).
I can't share a single app across the entire thing with `settings.multiMonitorFullScreen = true;` because of the different resolutions (projectors are 1080 high, monitors 1920).
I might be able to open a window for each display, but the projectors need a shared one because of the edge blending addon (plus, that might add some overhead? Not sure)

After a quick test using `ofxSecondWindow` with `BORDERLESS_NATIVE_SIZE` seems useful for creating two windows (1 for projectors, 1 for monitors) that I can easily position.

**Comment by danaavesar** [comment]
I agree I love this addon! It makes it much more simple to draw to two windows. I like how with this addon I can easily control what gets drawn to each window within a single app. I don't know how to easily control that with the new OpenFrameworks. Im sad that it doesnt work with the newest openFrameworks. Does anyone have tips for how to control what gets drawn on two windows in openFrameworks 9.0?
